packages/contractkit/src/contract-utils.ts

```diff
@@ -221,7 +221,8 @@ export async function sendTransactionAsync<T>(
   tx: TransactionObject<T>,
   account: string,
   gasCurrencyContract: StableToken | GoldToken,
-  logger: TxLogger = emptyTxLogger
+  logger: TxLogger = emptyTxLogger,
+  staticGas?: number | undefined
```

Comment:
Why not call this `estimatedGas` and eliminate ` let estimatedGas`?

Reply:
I'm not generally a fan of mutating function parameters, but added it (assuming that's what you are suggesting) 28bce37